airbyte-cdk/python/airbyte_cdk/sources/declarative/datetime/min_max_datetime.py

```diff
@@ -21,40 +21,49 @@ def __init__(
         datetime_format: str = "",
         min_datetime: Union[InterpolatedString, str] = "",
         max_datetime: Union[InterpolatedString, str] = "",
+        **options: Optional[Mapping[str, Any]],
     ):
+        self._datetime_interpolator = InterpolatedString.create(datetime, options=options)
+        self._datetime_format = datetime_format
+        self._timezone = dt.timezone.utc
+        self._min_datetime_interpolator = InterpolatedString.create(min_datetime, options=options) if min_datetime else None
+        self._max_datetime_interpolator = InterpolatedString.create(max_datetime, options=options) if max_datetime else None
         """
         :param datetime: InterpolatedString or string representing the datetime in the format specified by `datetime_format`
         :param datetime_format: Format of the datetime passed as argument
         :param min_datetime: InterpolatedString or string representing the min datetime
         :param max_datetime: InterpolatedString or string representing the max datetime
+        :param options: Additional runtime parameters to be used for string interpolation
         """
-        self._datetime_interpolator = InterpolatedString.create(datetime)
+        self._datetime_interpolator = InterpolatedString.create(datetime, options=options)
         self._datetime_format = datetime_format
         self._timezone = dt.timezone.utc
-        self._min_datetime_interpolator = InterpolatedString.create(min_datetime) if min_datetime else None
-        self._max_datetime_interpolator = InterpolatedString.create(max_datetime) if max_datetime else None
+        self._min_datetime_interpolator = InterpolatedString.create(min_datetime, options=options) if min_datetime else None
+        self._max_datetime_interpolator = InterpolatedString.create(max_datetime, options=options) if max_datetime else None
 
-    def get_datetime(self, config, **kwargs) -> dt.datetime:
+    def get_datetime(self, config, **additional_options) -> dt.datetime:
```

Comment:
was there a particular reason why we're changing `kwargs` to `additional_options` just for the MinMaxDatetime class, but not other components?

Reply:
this one is different because it's set in the `get_datetime`.
The `options` set in the `__init__` are named `options`